jprov/server/http.go

```diff
@@ -69,6 +69,7 @@ func checkVersion(cmd *cobra.Command, w http.ResponseWriter, ctx *utils.Context)
 	} else {
 		v = types.VersionResponse{
 			Version: version.Version,
+			ChainID: "test-chain",
```

Comment:
At this point, the "test-chain" ChainID string does appear even if the provider does not set a version flag.